packages/react-components/react-tree/src/components/Tree/Tree.types.ts

```diff
@@ -19,16 +19,17 @@ export type TreeNavigationData_unstable = {
   target: HTMLElement;
   value: TreeItemValue;
   parentValue: TreeItemValue | undefined;
-} & (
-  | { event: React.MouseEvent<HTMLElement>; type: 'Click' }
-  | { event: React.KeyboardEvent<HTMLElement>; type: 'TypeAhead' }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof ArrowRight }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof ArrowLeft }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof ArrowUp }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof ArrowDown }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof Home }
-  | { event: React.KeyboardEvent<HTMLElement>; type: typeof End }
-);
+} & FocusOptions &
```

Comment:
IMO `preventScroll` should be a function, there is no API in Fluent or Javscript that relies on mutating properties of callback arguments

Reply:
That is a valid point. I'll be updating the API to include methods instead of a `boolean` value